### ISSUE TITLE
1.0 No accessible extension method 'GetActionMap' accepting a first argument of type 'InputActionAsset' could be found

### ISSUE THREAD

**Issue by interior-night** [Unity-Technologies/InputSystem#872]
Getting this error with the latest 1.0.0

`Assets/Samples/Input System/1.0.0-preview/Simple Demo/SimpleControls.cs(151,28): error CS1061: 'InputActionAsset' does not contain a definition for 'GetActionMap' and no accessible extension method 'GetActionMap' accepting a first argument of type 'InputActionAsset' could be found (are you missing a using directive or an assembly reference?)`

Also getting that error for every other inputactionasset I have in my project.

**Comment by interior-night** [comment]
Having created a new InputActionAsset I'm noticing it has no errors but GetActionMap has been replaced by FindActionMap. If that's the issue, then it wouldn't hurt to communicate changes like these to your users. Especially when you bump to version 1.0.0. :-/ Plus you should probably communicate that to the StandardAssets folks (so that they can update their 'StandardControls') and to whoever did the Simple Demo that comes with the new Input System.

**Comment by harrison-drift** [comment]
Same issue here, fixed by editing my generated settings to replace all instances of Get with Find

**Comment by Rene-Damm** [comment]
The expectation here is that this is automatically handled by the asset import process. I.e. the asset db should realize that the asset importer version has changed and re-import .inputactions assets -- which in turn should update all generated .cs files. So, it *should* be resolved automatically.

Looking at the report here, though, it seems that this isn't happening. Not sure what's tripping it up along the way. Could be it's the compile errors that will temporarily result -- though the input system itself should still compile successfully and reload and then in turn regenerate the file and cause another reload.

Long story short, obviously room for improvement in our code generation pipeline :/

**Comment by stefanunity** [comment]
Hello,

we're going to close all the github issues and will concentrate bug information in the Unity public issue tracker https://issuetracker.unity3d.com/product/unity/issues?project=116&status=1&unity_version=&view=hottest.

If you are still experienceing this issue in the latest package version please do file a bug report as described here https://unity3d.com/unity/qa/bug-reporting
For general feedback and discussions you can go to https://forum.unity.com/forums/new-input-system.103/

Thank you!
